kasa/device.py

```diff
@@ -51,6 +53,15 @@ class Device(ABC):
     or :func:`Discover.discover_single()`.
     """
 
+    # All types required to create devices directly via connect are aliased here
+    # to avoid consumers having to do multiple imports.
+    Type: TypeAlias = DeviceType
+    Credentials: TypeAlias = Credentials
+    Config: TypeAlias = DeviceConfig
+    FamilyType: TypeAlias = DeviceFamilyType
+    EncryptionType: TypeAlias = EncryptType
```

Comment:
```suggestion
    EncryptType: TypeAlias = EncryptType
```
I think we should keep the naming consistent.

Reply:
So to keep naming consistent and simplify the aliases as per https://github.com/python-kasa/python-kasa/pull/936#discussion_r1623399959 I've updated the classes:

- `ConnectionType` > `DeviceConnection`
- `EncryptType` > `DeviceEncryption`
- `DeviceFamiltyType` > `DeviceFamily`

And deprecated the old names.  So these are now accessible under `Device` aliases, e.g. `Device.Family.SmartKasaPlug` and `Device.Encryption.Klap`